index.js

```diff
@@ -1,69 +1,272 @@
-  
-  menuLinkContactElement.addEventListener('click', () => {
-    if (window.screen.width < 768) {
-    menuElement.style.display = 'none';
-    menuIconElement.style.display = 'inline-block';
-    logoElement.style.display = 'inline-block';
-    navElement.style.width = 'fit-content';
-    navElement.style.height = '18px';
-    navElement.style.paddingTop = '0';
-    navElement.style.paddingBottom = '0';
+
+  projectCard.appendChild(headerElement);
+  projectCard.appendChild(descriptionElement);
+  projectCard.appendChild(stackList);
+  projectCard.appendChild(button);
+
+  return projectCard;
+}
+
+const projectsContainer = document.getElementById('project-cards-container');
+for (let a = 0; a < projectCardData.length; a += 1) {
+  const projectCard = createProjectCard(projectCardData[a]);
+  projectsContainer.appendChild(projectCard);
+}
+
+/* modal popup window elements */
+const projectButtonElements = document.querySelectorAll('.pjt-button');
+
+const createModalContentElements = (i) => {
+  const popupProjectName = document.createElement('h2');
+  popupProjectName.classList.add('modal-heading', 'header-text');
+  popupProjectName.textContent = `Project ${i + 1}`;
+  const popupStack1 = document.createElement('span');
+  popupStack1.classList.add('modal-stacks');
+  popupStack1.textContent = 'Html';
+  const popupStack2 = document.createElement('span');
+  popupStack2.classList.add('modal-stacks');
+  popupStack2.textContent = 'Bootstrap';
+  const popupStack3 = document.createElement('span');
+  popupStack3.classList.add('modal-stacks');
+  popupStack3.textContent = 'Ruby on Rails';
+  const div1 = document.createElement('div');
+  div1.classList.add('divide');
+  const div2 = document.createElement('div');
+  div2.classList.add('illustration');
+  const modalDescription = document.createElement('p');
+  modalDescription.classList.add('modal-description', 'description2-text');
+  modalDescription.textContent = 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent';
+  const popupButton1 = document.createElement('a');
+  popupButton1.classList.add('main-btn', 'see-live');
+  popupButton1.href = 'https://www.google.com/';
+  popupButton1.textContent = 'See live';
+  const popupButton2 = document.createElement('a');
+  popupButton2.classList.add('main-btn', 'see-source');
+  popupButton2.href = 'https://github.com/Reggeditt/Portfolio/';
+  popupButton2.textContent = 'See source';
+
+  return [
+    popupProjectName,
+    div1,
+    popupStack1,
+    popupStack2,
+    popupStack3,
+    div1,
+    div2,
+    modalDescription,
+    popupButton1,
+    popupButton2,
+    div1,
+  ];
+};
+
+const modalContainerElement = document.createElement('div');
```

Comment:
[OPTIONAL] Please follow the Figma template and fix the card. There should be three cards in one row. 👨‍🎨 Check the difference below.
Figma template ↓
<img width="853" alt="Screenshot 2023-01-07 at 7 22 09 PM" src="https://user-images.githubusercontent.com/25887253/211160814-93cae722-9052-4807-a079-24c982f38497.png">

![Screen Recording 2023-01-07 at 7 20 39 PM](https://user-images.githubusercontent.com/25887253/211160811-96a64826-9b5f-4e05-b0d3-4375e379c5ee.gif)